addons/udes_security/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
No reason to add a license as part of this work, and we've used the LGPL license for our other things